docs/reference/commandline/dockerd.md

```diff
@@ -828,6 +829,24 @@ To set the DNS search domain for all Docker containers, use:
 $ sudo dockerd --dns-search example.com
 ```
 
+#### Air-gapped registries
```

Comment:
@clnperez, @stevvooe: If we want to name for behavior (rather than use case) while avoiding "foreign layers", then "allow nonredistributable" feels like the right direction. How about one of these?